docs/docs/user-guide/connecting-office-365.md

```diff
@@ -69,7 +69,7 @@ Because there is no user context when logging in using a certificate, you will t
 Logging in using a certificate gives the CLI for Microsoft 365 app-only access to Microsoft 365 services. Not all operations support app-only access so it is possible, that some CLI commands will fail when executed while logged in to Microsoft 365 using a certificate.
 
 !!! attention
-    When logging in to Microsoft 365 using a certificate, CLI for Microsoft 365 will persist not only the retrieved access token but also the contents of the certificate's private key and its thumbprint. This is necessary for the CLI to be able to retrieve a new access token in case of the previously retrieved access token expired or has been invalidated.
+    When logging in to Microsoft 365 using a certificate, CLI for Microsoft 365 will persist not only the retrieved access token but also the contents of the certificate's private key. This is necessary for the CLI to be able to retrieve a new access token in case of the previously retrieved access token expired or has been invalidated.
```

Comment:
We should persist the calculate thumbprint so that we don't need to recalculate it on each command execution and can execute commands quicker.

Reply:
Ok